ocean_lib/ocean/v4/ocean_assets.py

```diff
@@ -78,7 +78,7 @@ def _add_defaults(
 
         if not has_access_service:
             access_service = self.build_access_service(
-                service_id="1",
+                service_id="0",
```

Comment:
why?

Reply:
I got an error from provider that the requested service needs to have the id set to 0 instead of 1.

Reply:
It was my bad when I first implemented this part.